client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1573,4 +1634,12 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     heartbeater.stop()
     super.stop()
   }
+
+  private def createSecret(): String = {
+    val bits = 256
+    val rnd = new SecureRandom()
+    val secretBytes = new Array[Byte](bits / JByte.SIZE)
+    rnd.nextBytes(secretBytes)
+    Hex.encodeHexString(secretBytes)
```

Comment:
The dependency on `commons-codec` is causing issues. Maybe we can just use bytes but this is how it is generated in Spark and I think that it is a common practice to convert secrets into hexadecimal format.

Reply:
reducing dependencies is generally a good idea, but please open a dedicated PR for it, also, SBT and LICENSE parts should be updated too

Reply:
ignore my previous comments, I misinterpret the message.